srml/indices/src/lib.rs

```diff
@@ -156,6 +156,21 @@ impl<T: Trait> Module<T> {
 }
 
 impl<T: Trait> OnNewAccount<T::AccountId> for Module<T> {
+	/// Implementation of the config type managing the creation of new accounts.
+	/// See `Balances` module for a concrete example.
+	///
+	/// # <weight>
+	/// - Independent of the arguments.
+	/// - Given the correct value of `Self::next_enum_set()`, it always has a limited
+	///   number of reads and writes and no complex computation.
+	///
+	/// As for storage, calling this function with _non-dead-indices_ will linearly grow the length of
+	/// of `Self::enum_set()`. Appropriate economic incentives should exist to make callers of this
+	/// function provide a `who` argument that reclaims a dead account.
+	///
+	/// At the time of this writing, only the balances module calls this function upon creation
```

Comment:
Maybe this comment is OK w/r/t Polkadot, but IMO we should not have comments like this in documentation. Substrate is a framework for people to use these modules and we have no idea if they will use this in some other way.
```suggestion
	/// At the time of this writing, only the Balances module calls this function upon creation
```

Reply:
Agreed, can be cleaned up.